frontend/src/components/HospitalInventoryEdit.module.css

```diff
@@ -0,0 +1,14 @@
+.inventoryInput {
+  /* width: 40%!important; */
+  text-align: center;
+}
+
+.inputCard {
+  border: none!important;
```

Comment:
Please don't use `!important` unless _absolutely necessary_. It's like the "GOTO of CSS". If you need something without a border, I'd advise avoiding `Card`s in the first place.